src/lib/y2storage/planned/md.rb

```diff
@@ -105,7 +105,7 @@ def self.to_string_attrs
       def volume_match_values
         {
           mount_point:  mount_point,
-          size:         DiskSize.zero,
+          size:         nil,
```

Comment:
Why don't we want to check the size of the Md devices? Doesn't `Md#size` return a sensible value just like e.g. `Partition#size`?

Reply:
It is a `Planned::Md` and it has not `#size`.